tests/saw/s2n_handshake_io.saw

```diff
@@ -20,8 +20,8 @@ let conn_secure_cipher_suite pconn =
 //let key_exchange_algorithm csuite = crucible_elem csuite 3;
 let key_exchange_algorithm csuite = crucible_field csuite "key_exchange_alg";
 
-//let kea_flags kea = crucible_elem kea 0;
-let kea_flags kea = crucible_field kea "flags";
+let kea_is_ephemeral kea = crucible_elem kea 0;
```

Comment:
I'm not too familiar with SAW but this definition of kea_is_ephemeral looks incorrect. Shouldn't this be using the crucible_field instead of an element?

Reply:
Attempting to use `let kea_is_ephemeral kea = crucible_field kea "is_ephemeral";` I was running into `Unable to resolve field name: "is_ephemeral"` errors on the travis build. This does not set kea_is_ephemeral to 0, instead it is a reference to the first element in the kea struct which is is_ephemeral.

Reply:
Attempting to use `let kea_is_ephemeral kea = crucible_field kea "is_ephemeral";` I was running into errors finding the symbol. This does not set kea_is_ephemeral to 0, instead it is a reference to the first element in the kea struct which is is_ephemeral.

See https://travis-ci.org/awslabs/s2n/jobs/453087828.